git_machete/cmd.py

```diff
@@ -1586,7 +1581,7 @@ def create_github_pr(self, head: str, draft: bool) -> None:
         debug(f'create_github_pr({head})', f'organization is {org}, repository is {repo}')
         debug(f'create_github_pr({head})', 'current GitHub user is ' + (current_user or '<none>'))
 
-        title: str = self.__git.popen_git("log", "-1", "--format=%s").strip()
+        title: str = self.__git.get_popen_git("log", "-1", "--format=%s").strip()
```

Comment:
As discussedm I'd stick to `popen_git` here (esp. that `popen` can be treated as kinda-verb here)